app/src/main/java/fr/free/nrw/commons/review/ReviewController.java

```diff
@@ -156,19 +156,23 @@ void sendThanks(@NonNull Activity activity) {
                 .subscribeOn(Schedulers.io())
                 .observeOn(AndroidSchedulers.mainThread())
                 .subscribe((result) -> {
-                    String message;
-                    String title;
-                    if (result) {
-                        title = context.getString(R.string.send_thank_success_title);
-                        message = context.getString(R.string.send_thank_success_message, media.getDisplayTitle());
-                    } else {
-                        title = context.getString(R.string.send_thank_failure_title);
-                        message = context.getString(R.string.send_thank_failure_message, media.getDisplayTitle());
-                    }
+                    displayToast(context,result);
+                }, Timber::e);
+    }
 
-                    showNotification(title, message);
+    @SuppressLint("StringFormatInvalid")
+    private void displayToast(final Context context, final boolean result){
```

Comment:
displayToast is too generic. I would suggest `displayThanksToast` as this function is specific to the use case. 
Also add some documentation to this function.